Gemfile

```diff
@@ -2,3 +2,7 @@ source 'https://rubygems.org'
 
 # Specify your gem's dependencies in interjectable.gemspec
 gemspec
+
+gem "pry"
```

Comment:
@zachmargolis You ok with adding `pry` to the `Gemfile` but not `require`-ing?

Reply:
Yup!